functions/Get-DbaSpn.ps1

```diff
@@ -48,20 +50,15 @@ Returns a custom object with SearchTerm (ServerName) and the SPNs that were foun
     [cmdletbinding()]
 	param (
         [Parameter(Mandatory = $false,ValueFromPipeline = $true)]
-        [string[]]$ComputerName,
+        [string[]]$ComputerName = $env:COMPUTERNAME,
         [Parameter(Mandatory = $false)]
 		[string[]]$AccountName,
+		[switch]$ShowAccountNameAll,
 		[Parameter(Mandatory = $false)]
         [PSCredential]$Credential
     )
 	process
-	{
-		if (!$ComputerName -and !$AccountName)
-		{
-			$ComputerName = $env:computername
-			$AccountName = "$env:USERDOMAIN\$env:USERNAME"
-		}
-		
+	{	
 		if ($ComputerName)
```

Comment:
true, i had to move it a bunch of stuff to ensure this still worked, please recheck my logic.

@drewfurgiuele - please check this PR and approve. I changed the functionality.